src/widget/wtrackmenu.cpp

```diff
@@ -949,6 +963,12 @@ void WTrackMenu::slotOpenInFileBrowser() {
     mixxx::DesktopHelper::openInFileBrowser(locations);
 }
 
+void WTrackMenu::slotFindOnSoundcloud(const Track& track) {
+    const auto artistName = track.getArtist();
```

Comment:
I think you can use     
`TrackPointer pTrack = getFirstTrackPointer();`
to obtain the track.

Reply:
That worked.

Thank you :)